src/analyzer/plugins/analyzerqueenmarybeats.cpp

```diff
@@ -9,17 +9,17 @@
 
 namespace mixxx {
 namespace {
-// These are the preferred window/step sizes from the BeatTrack VAMP plugin.
-constexpr size_t kWindowSize = 1024;
-constexpr size_t kStepSize = 512;
+// stepsize should be equivalent to ~10ms and this constant (11.61ms) make the number 
+// of samples an exact power of 2 for 22.1k, 44.1k, and very close to, for 96k and 192k
+constexpr float kStepSecs = 0.01161; 
```

Comment:
I'm confused because your [original commit](https://github.com/mixxxdj/mixxx/pull/2683/commits/4ed2e132693c3c3d286488daf400fb8559b45f27) made the step size 256, which would be 5.8 ms at 44.1 kHz. Why the change to 11.6 ms?

Reply:
Ok, I see this is the original step size:
https://github.com/mixxxdj/mixxx/blob/cb4a910bd4ee1a4cb561c4ff3fd6d338eae68794/vamp-plugins/plugins/BeatTrack.cpp#L28

I'm not sure where 256/512 came from in your original commit then?

Reply:
I was using Sonic Visualizer to make some tests and it was the default values for the 22100 tracks I was testing. And by then I did not know yet that this was sample rate dependent, so I thought that was the default value for the vamp plugin.